packages/beacon-node/test/unit/chain/validation/attestation/validateAttestation.test.ts

```diff
@@ -3,7 +3,6 @@ import {describe, expect, it} from "vitest";
 import {ForkName, SLOTS_PER_EPOCH} from "@lodestar/params";
 import {ssz} from "@lodestar/types";
 import {LodestarError} from "@lodestar/utils";
-// eslint-disable-next-line import/no-relative-packages
```

Comment:
we need ot be careful with those imports as it would break production code if done there, the lint rule seems important

Reply:
I added an issue to bring this behavior back. https://github.com/ChainSafe/lodestar/issues/7137